tools/clang/lib/SPIRV/CapabilityVisitor.cpp

```diff
@@ -559,8 +559,8 @@ bool CapabilityVisitor::visitInstruction(SpirvInstruction *instr) {
       addCapability(spv::Capability::RayTracingKHR);
       addExtension(Extension::KHR_ray_tracing, "SPV_KHR_ray_tracing", {});
     }
+    break;
```

Comment:
This looks like a behavior change... possibly a correct change, but a behavior change.

Reply:
It is. I'm confident it's correct. I'll ask @keenuts or @cassiebeckley to take a look. I see no reason why RT ops should fallthrough to mesh shader handling.

Reply:
Yes, that's a behavior change, but a good one 🙃
This was a bug.